app/models/event.rb

```diff
@@ -3,6 +3,7 @@ class Event < ApplicationRecord
   belongs_to :user
   has_and_belongs_to_many :members, class_name: "User"
   accepts_nested_attributes_for :place
+  default_scope {order(started_at: :desc)}
```

Comment:
1. I don't recommend you default scope. In most cases this is magic why elements are in specific order or some of them are not visible. In my opinion is better to use normal scope and use it directly where you need it. When you have default scope you model element will always use it and removing default scope can provide in future more problems.
2. If you already created default scope why you don't use it?

Reply:
Please create normal scope like we discuss it some time ago with the name we choose.

Reply:
```
  def index
    @events = Event.all
    @events.order('started_at DESC')
  end
``` 

and without 
```
  default_scope { order(started_at: :desc) }
```
in controller???

Reply:
I added comment to our previous conversation.